lib/Rex/CLI.pm

```diff
@@ -517,8 +519,15 @@ CHECK_OVERWRITE: {
 
     my %groups = Rex::Group->get_groups;
     _print_color( "Server Groups\n", "yellow" ) if ( keys %groups );
+    my ($cols) = Term::ReadKey::GetTerminalSize(*STDOUT);
+    $Text::Wrap::columns = $cols // 80;
```

Comment:
Hi @kablamo ,

the pr fails because the *//* operator is not available in perl 5.8. But for Rex 1.x we want to keep the perl 5.8 compatibility. So i think this line can also be written like this:

```perl
$Text::Wrap::columns = $cols || 80;
```